ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/PullQueryExecutorMetrics.java

```diff
@@ -83,18 +84,24 @@ public void recordRemoteRequests(final double value) {
     this.remoteRequestsSensor.record(value);
   }
 
-  public void recordLatency(final long startTimeNanos) {
+  public void recordLatency(final double value) {
     // Record latency at microsecond scale
-    final long nowNanos = time.nanoseconds();
```

Comment:
Why did you remove how this was being done by encapsulating the nowNanos?  It prevented you from having to do that all over.

Reply:
I don't understand what you mean here :(

Reply:
Added it back, you are right